maxscale-system-test/mdbci/templates/default.json.template

```diff
@@ -104,7 +104,19 @@
         }
   },
 
-  "maxscale" :
+  "maxscale_000" :
+  {
+        "hostname" : "maxscale",
+        "box" : "${box}",
+        "memory_size" : "${vm_memory}",
+        "product" : {
+                 "name" : "maxscale_ci",
+                "version" : "${target}"
+        }
+
+  },
+
+  "maxscale_001" :
   {
         "hostname" : "maxscale",
```

Comment:
Both have the same hostname, might cause problems. Changing it to `maxscale2` would be OK.